### ISSUE TITLE
Uncaught TypeError: Cannot read property 'hasAttribute' of null

### ISSUE THREAD

**Issue by kavinmehta** [adobe/aem-guides-wknd#124]
Hi,
I downloaded & installed the latest package & source code of WKND (0.0.4) on my local and after npm install on source code & running storybook I keep getting "Uncaught TypeError: Cannot read property 'hasAttribute' of null" error in console and accordion does not work for me.

Anyone else getting the same error?

![image](https://user-images.githubusercontent.com/142260/89718274-4f68f880-d972-11ea-9934-7cccf05c7e5d.png)
![image](https://user-images.githubusercontent.com/142260/89718292-5b54ba80-d972-11ea-936d-98933e19c445.png)

Thanks,
Kavin Mehta.

**Comment by godanny86** [comment]
Hey Kavin,

What version of AEM, service packs (if any) are you using?

Daniel Gordon | dgordon@adobe.com | AEM Tech Marketing | 571.236.2970
________________________________

**Comment by godanny86** [comment]
Sorry just rereading this. You are seeing this error when running storybook? It probably got introduced when the adobe data layer was added...

**Comment by kavinmehta** [comment]
Hi,

I am running AEM 6.5.3 on Windows but yes it's because of adobe data layer. The accordion js script is looking for 'data-cmp-data-layer-enabled' in the body tag but Storybook doesn't have it so it fails.  It looks like all core components that need JS to render are now looking for data-cmp-data-layer-enabled in body tag. I tried injecting it using preview-body.html in storybook but it didn't work. The core component js script needs to do some type of error handling maybe.

Thanks,
Kavin Mehta.

**Comment by godanny86** [comment]
Yeah I have had issues injecting external javascript using preview-body.html in storybook as well. Will have to investigate further. If you come up with something feel free to submit a PR

**Comment by chaithanyaJaishankar** [comment]
Hi Team,

Even I am facing same error as @kavinmehta has shared earlier, for me this is breaking Image component showing no images, using aem-sdk Adobe Experience Manager 2020.8.4139.20200830T210742Z-200827.

Thanks,

Chaithanya

**Comment by d3zza** [comment]
I am getting this error too. FWIW I don't think this error is to do with the Storybook or the presence of 'data-cmp-data-layer-enabled' but that the **body** is null when this code is executed. Seems like the code is in an IIFE and executing before DOM is ready.
